tests/functions/module_custom.test.ts

```diff
@@ -130,3 +130,15 @@ describe("FORMAT.LARGE.NUMBER formula", () => {
     expect(evaluateCellText("A1", { A1: '=FORMAT.LARGE.NUMBER(100, "something")' })).toBe("#ERROR");
   });
 });
+
+describe("HYPERLINK formula", () => {
+  test("The evaluated result is the markdown format of link", () => {
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com", "Odoo")' })).toBe(
+      "Odoo"
+    );
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com")' })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("invalidUrl")' })).toBe("invalidUrl");
```

Comment:
test with
- reference `evaluateCell("A1", { A1: '=HYPERLINK(A2)', A2: "https://www.odoo.com" })`
- empty string `evaluateCellText("A1", { A1: '=HYPERLINK("")' })` and `evaluateCellText("A1", { A1: '=HYPERLINK("  ")' })`
- a number `evaluateCellText("A1", { A1: '=HYPERLINK(2)' })`
- a boolean `evaluateCellText("A1", { A1: '=HYPERLINK(true)' })`
- a date `evaluateCellText("A1", { A1: '=HYPERLINK("1/31/2022")' })`
- no argument `evaluateCellText("A1", { A1: '=HYPERLINK()' })`